src/v3/src/transformer/webauthn/transformWebAuthNAuthenticator.ts

```diff
@@ -26,42 +32,123 @@ import {
   webAuthNEnrollmentHandler,
 } from '../../util';
 
-const SUPPORTED_STEPS = [IDX_STEP.ENROLL_AUTHENTICATOR, IDX_STEP.CHALLENGE_AUTHENTICATOR];
+const appendViewCallouts = (
```

Comment:
This was missing for parity: 
**Enroll**
https://github.com/okta/okta-signin-widget/blob/0.0/src/v2/view-builder/views/webauthn/EnrollWebauthnInfoView.js#L9-L25
**Challenge**
https://github.com/okta/okta-signin-widget/blob/0.0/src/v2/view-builder/views/webauthn/ChallengeWebauthnInfoView.js#L8-L17